static/_redirects

```diff
@@ -156,4 +156,7 @@ docs/understanding-tracking-design/managing-data-structures-with-data-structures
 
 /docs/getting-started-on-snowplow-open-source/setup-snowplow-on-aws/* /docs/getting-started-on-snowplow-open-source/quick-start-aws/ 301
 /docs/getting-started-on-snowplow-open-source/setup-snowplow-on-gcp/* /docs/getting-started-on-snowplow-open-source/quick-start-gcp/ 301
-/docs/getting-started-on-snowplow-open-source/what-is-quick-start/faq/ /docs/getting-started-on-snowplow-open-source/faq/ 301
\ No newline at end of file
+/docs/getting-started-on-snowplow-open-source/what-is-quick-start/faq/ /docs/getting-started-on-snowplow-open-source/faq/ 301
+
+/docs/tutorials/tutorial-a-guide-to-using-the-snowplow-inspector-extension/* /docs/testing-debugging/snowplow-inspector/:splat 301
+/docs/getting-started-with-micro/* /docs/testing-debugging/snowplow-micro/:splat 301
```

Comment:
This should work for the parent page as Nelify will not redirect for pages that exist